Library/Location.swift

```diff
@@ -3,6 +3,8 @@ import CoreLocation
 
 struct Location {
 
+    private let calendar = Calendar.current
```

Comment:
why `current` instead of `autoupdatingCurrent`?

Reply:
Was checking if it made a difference. It's never alive long enough to matter, IMO. But I'm happy to revert back if you prefer.